src/js/firstparties/facebook.js

```diff
@@ -51,29 +34,11 @@ function cleanLink(a) {
   a.addEventListener("mouseover", function (e) { e.stopPropagation(); }, true);
```

Comment:
Should we also call `e.stopImmediatePropagation`?

Reply:
Yes, definitely.